doc/GUIDE.md

```diff
@@ -835,6 +843,30 @@ can also specify additional clang compiler options, by creating a file named
 `compile_flags.txt` in the current folder, as described
 [here](https://clang.llvm.org/docs/JSONCompilationDatabase.html#alternatives).
 
+## Unsupported Features
+
+There are several elements for which jextract can not generate bindings:
```

Comment:
element seems a bit vague, maybe "C program elements" could be better, although in this cases I suppose most developers will think about the term "features". Consider just having a bullet list and skip the first statement (after all, the title says it all). Note also the list is provided here:
https://jdk.java.net/jextract/
(maybe we should drop it from there in the future, to avoid duplication?)